### PR TITLE
Add Korean(한국어)

### PR DESCRIPTION
**Pull request by DevNergis**
# 한국어, Korean, ko_KR

screencapture-localhost-4321-2024-07-21-00_06_03

### CODE REVIEW

**Comment by saicaca** [comment]
Thank you for your contribution!

File: src/i18n/languages/ko_KR.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import Key from '../i18nKey'
+import type { Translation } from '../translation'
+
+export const ko_KR: Translation = {
```
Comment (saicaca) [inline src/i18n/languages/ko_KR.ts:4]: I think renaming this to `ko` would be better, making it consistent with `en` and `jp`.

Comment (DevNergis) [reply src/i18n/languages/ko_KR.ts:4]: ok

**Comment by DevNergis** [comment]
# ko_KR to ko

**Comment by DevNergis** [comment]
fix README

File: package-lock.json
Comment (saicaca) [inline package-lock.json:1]: Could you please remove this file? I prefer to use only pnpm for this project.

Comment (DevNergis) [reply package-lock.json:1]: ok

**Comment by DevNergis** [comment]
remove package-lock.json and update .gitignore
```
package-lock.json
bun.lockb
```

**Comment by saicaca** [comment]
LGTM!

**Comment by DevNergis** [comment]
Thx!

**Pull request merged**